qiskit/finance/__init__.py

```diff
@@ -13,7 +13,7 @@
 # that they have been altered from the originals.
 """
 ==========================================================
-Finance application stack for Aqua (:mod:`qiskit.finance`)
```

Comment:
I would like to keep this high level description consistent across the other stacks too. Presently all the others say 'XXXX application stack for Aqua' since the intent was to provide higher level applications.